### DISCUSSION TITLE
Two-component spinor calculations in FeynCalc

### DISCUSSION THREAD

**Discussion by mg18121** [FeynCalc/feyncalc#232] [Q&A]
Hello! I am very new to FeynCalc and I am wondering if FeynCalc is suitable for calculations using two-component spinor techniques described [arXiv:0812.1594](https://arxiv.org/abs/0812.1594). In particular, I have the following questions:
1. Is it possible to define $\overline{\sigma} = \left(\mathbf{1},-\vec{\sigma}\right)$ in addition to PauliSigma, which is $\sigma = \left(\mathbf{1},\vec{\sigma}\right)$?
2. Is it possible to use FeynCalc to evaluate the traces of the following form (see Eqs. (2.56-2.58) in [arXiv:0812.1594](https://arxiv.org/abs/0812.1594)):

$\text{Tr}\left[\sigma^\mu \overline{\sigma}^\nu\right] = 2 g^{\mu \nu}$,
$\text{Tr}\left[\sigma^{\mu}\overline{\sigma}^\nu \sigma^\rho \overline{\sigma}^{\kappa}\right] = 2\left(g^{\mu\nu}g^{\rho \kappa} - g^{\mu \rho} g^{\nu \kappa} + g^{\mu \kappa}g^{\nu \rho} + i \epsilon^{\mu\nu\rho\kappa}\right)$?

3. Is it possible to introduce dotted and undotted indecies?

Any clarifications are greatly appreciated!

**Comment by vsht** [comment]
Hi,

the Pauli algebra implemented in FeynCalc is essentially
geared towards nonrelativistic calculations rather than SUSY.

All the points you mentioned would require quite some
changes in the source code. So if you need something that works
out-of-the-box, I'm afraid that you need to look for another tool.

Cheers,
Vlad


Am 20.09.23 um 01:06 schrieb mg18121:

**Reply by mg18121** [reply]
Got it! Thank you!
